recipe/meta.yaml

```diff
@@ -24,12 +24,11 @@ requirements:
   run:
     - python
     - pickleshare
-    - simplegeneric >0.8
+    - python-simplegeneric >0.8
```

Comment:
Bump the build number? Or we don't have the "other" `simplegeneric` as a feestock yet?